src/flow-private.h

```diff
@@ -74,7 +74,7 @@ enum {
     FLOW_PROTO_MAX,
 };
 /* max used in app-layer (counters) */
-#define FLOW_PROTO_APPLAYER_MAX FLOW_PROTO_UDP + 1
+#define FLOW_PROTO_APPLAYER_MAX (FLOW_PROTO_UDP + 1)
```

Comment:
nit: shouldn't this one be part of https://github.com/OISF/suricata/pull/11701/commits/c32bc616ee5bd436c1a58c708b14bce841f12956 :P

Reply:
Both are possible.
As a matter of facts, I did https://github.com/OISF/suricata/commit/c32bc616ee5bd436c1a58c708b14bce841f12956 after finding this parenthesis addition was needed for 897b6606e4ae116337ce9899d34c7ab9c6a14fda